src/ORM/Hierarchy/Hierarchy.php

```diff
@@ -421,16 +394,23 @@ public function showingCMSTree()
      *
      * @param bool $showAll Include all of the elements, even those not shown in the menus. Only applicable when
      *                      extension is applied to {@link SiteTree}.
+     * @param bool $IDLess Set to true to supress the ParentID and ID where statements.
      * @return DataList
      */
-    public function stageChildren($showAll = false)
+    public function stageChildren($showAll = false, $IDLess = false)
```

Comment:
I would probably call this something like`$skipParentIDFilter`.

I would also note that this won't work if people have overridden stageChildren, but the original code only calls stageChildren on subclasses, so that's no worse than my implementation.

Reply:
Done.